sdk/deviceupdate/azure-iot-deviceupdate/src/test/java/com/azure/iot/deviceupdate/UpdatesClientTests.java

```diff
@@ -32,6 +30,10 @@ private UpdatesAsyncClient createClient() {
         TokenCredential credentials;
         HttpClient httpClient;
         HttpPipelinePolicy recordingPolicy = null;
+        HttpPipeline httpPipeline;
+
+        HttpHeaders headers = new HttpHeaders().put("Accept", ContentType.APPLICATION_JSON);
+        AddHeadersPolicy addHeadersPolicy = new AddHeadersPolicy(headers);
```

Comment:
A lot of this looks duplicated, in the future could we make a base test class?

Reply:
It will be updated in a follow-up PR